src/device-registry/models/Device.js

```diff
@@ -112,6 +112,13 @@ const deviceSchema = new mongoose.Schema(
       trim: true,
       lowercase: true,
     },
+    device_codes: [
```

Comment:
the new field introduced that will assist in the data pipeline works when adding new devices to our networks.

We could also use this same PR to make the device_number field optional----in support for our international expansion works? @NoahNsimbe